internal/pkg/cli/env_delete.go

```diff
@@ -31,13 +32,17 @@ import (
 const (
 	envDeleteAppNameHelpPrompt = "An environment will be deleted in the selected application."
 	envDeleteNamePrompt        = "Which environment would you like to delete?"
-	fmtDeleteEnvPrompt         = "Are you sure you want to delete environment %s from application %s?"
+	fmtDeleteEnvPrompt         = "Are you sure you want to delete environment %q from application %q?"
 )
 
 const (
-	fmtDeleteEnvStart    = "Deleting environment %s from application %s."
-	fmtDeleteEnvFailed   = "Failed to delete environment %s from application %s.\n"
-	fmtDeleteEnvComplete = "Deleted environment %s from application %s.\n"
+	fmtRetainEnvRolesStart    = "Retain IAM roles before deleting the %q environment"
+	fmtRetainEnvRolesFailed   = "Failed to retain IAM roles for the %q environment\n"
+	fmtRetainEnvRolesComplete = "Retained IAM roles for the %q environment\n"
+
+	fmtDeleteEnvStart    = "Delete IAM roles and deregister environment %q from application %q."
+	fmtDeleteEnvFailed   = "Failed to deregister environment %q from application %q.\n"
+	fmtDeleteEnvComplete = "Deregistered environment %q from application %q.\n"
```

Comment:
i feel like after typing `env delete` i'd expect it to say the environment was deleted...
```suggestion
	fmtDeleteEnvComplete = "Deleted environment %q from application %q.\n"
```

Reply:
👍 I also split up the Failed messages into two